go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	github.com/HewlettPackard/dws v0.0.0-20230410193930-8fd6fbeff7f8
 	github.com/NearNodeFlash/lustre-fs-operator v0.0.0-20230322153812-315a443f315b
-	github.com/NearNodeFlash/nnf-ec v0.0.0-20230425135615-c60e6e460894
+	github.com/NearNodeFlash/nnf-ec v0.0.0-20230426152614-be78544c8f4a
```

Comment:
Let's be careful about pointing into branches that are going to be deleted.  That can cause problems for someone who's trying to vendor on their own laptop, where didn't previously cache that branch.

Reply:
I always re-vendor before merging to point at the commit in master. I just put out all the pull requests at once so you can see all the parts.